src/libstore/filetransfer.hh

```diff
@@ -50,8 +50,9 @@ struct FileTransferSettings : Config
 
     Setting<size_t> downloadBufferSize{this, 64 * 1024 * 1024, "download-buffer-size",
         R"(
-          The size of Nix's internal download buffer during `curl` transfers. If data is
+          The size of Nix's internal download buffer in bytes during `curl` transfers. If data is
           not processed quickly enough to exceed the size of this buffer, downloads may stall.
+          The default is 67108864 (64 MiB).
```

Comment:
Do we really need this extra line? Since `nix.conf` already shows the default:

```
       •  download-buffer-size

          The  size  of  Nix’s  internal  download  buffer during curl transfers. If data is not processed quickly enough to exceed the size of this buffer, downloads may
          stall.

          Default: 67108864
```